src/TextForEvent.js

```diff
@@ -248,6 +248,26 @@ function textForPowerEvent(event) {
     });
 }
 
+function textForWidgetEvent(event) {
+    const senderName = event.sender ? event.sender.name : event.getSender();
+    const previousContent = event.getPrevContent() ? event.getPrevContent() : {};
+    const {name, type, url} = event.getContent() ? event.getContent() : {};
+    let widgetName = widgetName || name || type || previousContent.type || '';
+
+    // Apply sentence case
+    widgetName = widgetName ? widgetName[0].toUpperCase() + widgetName.slice(1).toLowerCase() + ' ' : '';
```

Comment:
This is the same logic applied to the app tile header, I was going for consistency. Ideally the name would be formatted as intended for human reading by Scalar itself.